src/components/views/elements/AppTile.tsx

```diff
@@ -590,7 +590,11 @@ export default class AppTile extends React.Component<IProps, IState> {
         const iframeFeatures =
             "microphone; camera; encrypted-media; autoplay; display-capture; clipboard-write; " + "clipboard-read;";
 
-        const appTileBodyClass = "mx_AppTileBody" + (this.props.miniMode ? "_mini  " : " ");
+        const appTileBodyClass = classNames({
+            mx_AppTileBody: !this.props.miniMode,
```

Comment:
`mx_AppTileBody` is always is present in the previous version

Reply:
No, because `"mx_AppTileBody" + (this.props.miniMode ? "_mini` outputs `"mx_AppTileBody_mini` on `miniMode`.

If miniMode is not enabled, `mx_AppTileBody ` (mind the last white space character) is output.

Here is the how the class name is output on develop.element.io:

![1](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/367dbca4-071d-4ca5-b54d-04206e8f68f2)

Reply:
I updated the table with "before" pictures above: https://github.com/matrix-org/matrix-react-sdk/pull/10939#issue-1715491853

Reply:
You're right